### ISSUE TITLE
Generate certificates of completion

### ISSUE THREAD

**Issue by kurtisnelson** [jah2488/classroom#131]
We should be able to generate a PDF for a whole cohort at once.

**Comment by jah2488** [comment]
Good idea. Never thought of this, because the CD has always handled it.

**Comment by kurtisnelson** [comment]
*currently saving out 12 PDFs to dropbox for them*

**Comment by jwo** [comment]
I'd love to write reports and email to them.

**Comment by jah2488** [comment]
@jwo go go go go go :yellow_heart:

**Comment by dorton** [comment]
I'll take on trying to build certificates if okay with y'all. Cause boo to non-automated pdf whatevers.

**Comment by kurtisnelson** [comment]
Here's the existing template: https://www.dropbox.com/s/z30un1stmfp20og/CertificateofCompletion_Form-NASHRoR.pdf?dl=0
